packages/build/src/steps/error.js

```diff
@@ -50,6 +55,12 @@ export const handleStepError = function ({
   }
 
   if (type === 'cancelBuild') {
+    const cancellationAttributes = {
+      'build.cancellation.title': title,
+      'build.cancellation.message': message,
+      'build.cancellation.packageName': packageName,
+    }
+    addEventToActiveSpan('build.cancelled', cancellationAttributes)
```

Comment:
Just adding a mental note that we'll want to test this out once it lands in prod 👌